### PR TITLE
Add episode trigger

### PR DESCRIPTION
**Pull request by tgromad**
Added Episode trigger to check for duplicates in same season

### CODE REVIEW

File: force-app/main/default/classes/TestDuplicateEpisode.cls [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        String errorM = '';
+        Test.startTest();
+        try {
+        insert episodeTwo;         
+            }
+        catch(Exception e){
+            errorM = e.getMessage();
+        }
+        Test.stopTest();
+        System.assertNotEquals('', errorM);
```
Comment (mlrogala) [inline force-app/main/default/classes/TestDuplicateEpisode.cls]: If insert fails the error message should be as you set it in trigger ('There is already an episode of the same name this season!')

File: force-app/main/default/classes/TestDuplicateEpisode.cls [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        Test.startTest();
+        episodeTwo.Episode_Name__c = 'name';
+        try {
+        update episodeTwo;
+        }
+        catch(Exception e){
+            errorM = e.getMessage();
+        }
+        Test.stopTest();
+        System.assertNotEquals('', errorM);
```
Comment (mlrogala) [inline force-app/main/default/classes/TestDuplicateEpisode.cls]: If insert fails the error message should be as you set it in trigger ('There is already an episode of the same name this season!')

**Review by mlrogala — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/TestDuplicateEpisode.cls (hunk 1 above)
Comment (tgromad) [reply force-app/main/default/classes/TestDuplicateEpisode.cls]: Done

File: force-app/main/default/classes/TestDuplicateEpisode.cls (hunk 2 above)
Comment (tgromad) [reply force-app/main/default/classes/TestDuplicateEpisode.cls]: Done

**Review by mlrogala — APPROVED** [review]

**Pull request merged**